jekyll/_cci2/testing-orbs.md

```diff
@@ -17,226 +17,276 @@ This guide covers various best practices for testing orbs.
 ```
 
+Want to see how how CircleCI writes unit tests for Bash? Check out our [Slack orb](https://github.com/CircleCI-Public/slack-orb/blob/master/src/tests/notify.bats).
+
 ## Integration testing
 {: #integration-testing }
 
-Up until this point, all testing has happened prior to packing the orb, and is applied to the code itself, not the finalized functioning orb. For the final, critical part of orb testing, you will test your orb commands and jobs to ensure they work as intended in production. This happens after the validation tests have run and a new development version of your orb is published.
-
-After the development version of your orb has been published, the [integration-test_deploy](https://github.com/CircleCI-Public/Orb-Project-Template/blob/master/.circleci/config.yml#L78) workflow will be triggered automatically to test it.
-
-The `integration-test_deploy` workflow runs a series of final integration tests, and if all pass, and you are on your main deployment branch, you can deploy your orb.
+After validating, linting, shellchecking, and any other testing that you can perform on the source code is complete, we must test our orb's functionality in a real CircleCI config. In the second config file (`test-deploy.yml`), we will access the development version of the orb we published in the first config, and attempt to execute our orbs commands and jobs.
 
 ### Testing orb commands
 {: #testing-orb-commands }
 
-The first job you will see in the [integration-test_deploy](https://github.com/CircleCI-Public/Orb-Project-Template/blob/master/.circleci/config.yml#L78) workflow is the [integration-test-1](https://github.com/CircleCI-Public/Orb-Project-Template/blob/master/.circleci/config.yml#L82) job, a sample integration test included with the `hello-world` orb, generated by the `orb-init` command.
+By default, when you author a new orb, you will have an example orb source which comes with a "greet" command. We want to test the greet command (and maybe other commands) in our `test-deploy` workflow as an integration test. We'll be able to execute the commands to validate they run without error, and we could even verify their functionality by running additional checks.
+
+You should see a job by default in your `test-deploy.yml` file named `command-tests`. This example job will run one or all of our commands as an integration tests.
 
-You can see the definition of the [`integration-test-1` job](https://github.com/CircleCI-Public/Orb-Project-Template/blob/master/.circleci/config.yml#L27) above in the `jobs` key.
+In it, we will call our orb command, with any parameters we want to test. If our command were to do something such as, installing a command line tool, we can then test to ensure that command is valid in an additional step.
 
+By default you will see the included "greet" command is being tested. Because the greet command only outputs a message to stdout, we can not do any additional validation checks.
```

Comment:
```suggestion
By default you will see the included "greet" command is being tested. Because the greet command only outputs a message to stdout, you can not do any additional validation checks.
```